### DISCUSSION TITLE
[solved] When it's time to go with filedb, server and lowdb?

### DISCUSSION THREAD

**Discussion by atherdon** [atherdon/dashb#45] [Ideas]
@vadim9999 what do you think about it?

https://github.com/typicode/lowdb

Right now I only with quick results. Bad code, etc. But it will help me to show our progress to our dev team.

From the other side - at some point we'll need to make it work normally, so I think we should consider to connect to lowdb at some point in next months

**Comment by vadim9999** [comment]
Hello

**Reply by vadim9999** [reply]
1

**Comment by atherdon** [comment]
Ok, after checking few projects/frameworks, I realize that a lot of them just add so much complexity.
And I still lazy to handle that complexity. Even if it's better, secure, etc.

But it's giving me more inspiration to try lowdb. Because at some point we'll be stuck on uploading data.

let's explore more, before just trying to implement it. And decide it together.

sample: https://stackoverflow.com/questions/54325446/how-do-i-connect-lowdb-in-react-project

**Reply by atherdon** [reply]
thinking about applying some sort of database. but lazy to accept that we need to have a normal nodejs server and also stuck later with that stack.

after checking different frameworks and tools I still interested to try lowdb.
exploring other things as well

but lowdb has a memory database and looks very easy to read from json files as well. https://github.com/typicode/lowdb#reading-from-existing-json-file

here is another way to use it: https://jsonbin.io/

**Comment by atherdon** [comment]
going with graphql backend
